src/components/dataset/query/DatasetQueryView.jsx

```diff
@@ -213,7 +213,7 @@ export class DatasetQueryView extends React.PureComponent {
               <QueryViewDropdown options={names} onSelectedItem={this.handleChange} />
             </Grid>
             <Grid item xs={3}>
-              <Link to="details">
+              <Link to="overview">
```

Comment:
huh...I thought we fixed this already

Reply:
We did! Just updated with contents from develop